src/lib.rs

```diff
@@ -120,24 +159,19 @@ impl TryFrom<&str> for Personnummer {
             date,
             serial,
             control,
-            divider,
-            coordination: (day > 31),
+            seperator,
+            coordination: (day / COORDINATION_NUMBER) >= 1,
         })
     }
 }
 
 impl Personnummer {
-    /// Returns a new instance of a [Personnummer]. Panics for invalid dates but not for invalid
-    /// personal identity numbers. Use [Personnummer::valid()] to check validity.
+    /// Returns a new instance of a [Personnummer]. Returns an error for invalid dates but not
+    /// for invalid personal identity numbers. Use [Personnummer::valid()] to check validity.
     pub fn new(pnr: &str) -> Result<Personnummer, PersonnummerError> {
         Personnummer::try_from(pnr)
     }
 
-    /// Same as [Personnummer::new()] but returns an [Option] instead of panicing on invalid dates.
-    pub fn parse(pnr: &str) -> Result<Personnummer, PersonnummerError> {
-        Personnummer::try_from(pnr)
-    }
-
     /// Returns a [FormattedPersonnummer] from a [Personnummer] which can be used to display a
     /// normalized version of the [Personnummer].
     pub fn format(&self) -> FormattedPersonnummer {
```

Comment:
Can we add a link to https://github.com/personnummer/meta/tree/master?tab=readme-ov-file#short-format and/or https://github.com/personnummer/meta/tree/master?tab=readme-ov-file#long-format while we're fixing this?